pallets/jobs/src/functions.rs

```diff
@@ -352,26 +352,14 @@ impl<T: Config> Pallet<T> {
 		// Validate existing result
 		ensure!(phase_one_result.ttl >= now, Error::<T>::ResultExpired);
 
-		// ensure the participants are the expected participants from job
-		let mut participant_keys: Vec<sp_core::ecdsa::Public> = Default::default();
-
-		let participants = phase_one_result.participants().ok_or(Error::<T>::InvalidJobPhase)?;
-		for participant in participants {
-			let key = T::RolesHandler::get_validator_role_key(participant);
-			ensure!(key.is_some(), Error::<T>::ValidatorRoleKeyNotFound);
-			let pub_key = sp_core::ecdsa::Public::from_slice(&key.expect("checked above")[0..33])
-				.map_err(|_| Error::<T>::InvalidValidator)?;
-			participant_keys.push(pub_key);
-		}
```

Comment:
Why this logic is removed?

Reply:
Also you're querying the session key and chcecming if it's a valid ECDSA. This all feels redundant, how would the key not be an ECDSA? The session key system would throw an error long before this is an issue in this pallet.